packages/design-system/src/components/FieldContainer/FieldContainer.tsx

```diff
@@ -82,11 +97,12 @@ export class FieldContainer extends React.Component<FieldContainerProps> {
     }
   }
 
-  render() {
+  render(): React.ReactNode {
```

Comment:
Is this return type necessary? I've never had to annotate this before

Reply:
I was getting an typescript warning on it, maybe my typescript config is wonky. I was getting other weird typescript errors too that i need to look into